### PR TITLE
Geo-timeline

### PR DESCRIPTION
**Pull request by mgallegos3015**
This is the first part (the 1930s) of the timeline. It's not completely presentable- I'm not sure in which format to cite the sources and some of the data I used for the maps has an attribution and share-alike license (https://creativecommons.org/licenses/by-sa/3.0/) that I wasn't sure how to attribute, so I'm pull requesting for some feedback. Additionally, on my machine the temporary teaser picture I was using keeps showing up instead of the new one I added, which I'm confused by since that picture no longer exists in the files, so let me know what you see.

### CODE REVIEW

File: templates/archives/stories/geo_timeline.jinja2 [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+</figure>
+    </div>
+    <p>Blue dot- Berlin, Germany, 1939- Konrad Zuse develops the Z2. This computer was relay
+        based as an improvement on the Z1, which did not work very well. It was almost done in 1939, but it was delayed to 1940 due to Zuse being required to join the army.</p>
+    <p>Red dot- New York City, United States, 1939- George Stibitz produces the Model I Complex
+        Calculator, the first electrical digital computer. It built on the boolean logic circuits Stibitz created. This computer utilized telephone sitches and electromechanical relays.</p>
+
+    <p></p>
+    <div class="text-center">
+<h1>Sources</h1>
```
Comment (ltagliaferri) [inline templates/archives/stories/geo_timeline.jinja2]: In the sources I would add which ones are CC, you can just say CC BY-SA 3.0 in parentheses next to the relevant sources with a link to the URL you shared with me (https://creativecommons.org/licenses/by-sa/3.0/)

File: apps/archives/views.py [hunk 2]
```diff
@@ -24,6 +24,7 @@
     'three_man_months',
     'women_in_symbols',
     'whirlwind',
+    'geo_timeline',
```
Comment (ltagliaferri) [inline apps/archives/views.py]: Make sure this does not overwrite other views

File: templates/archives/stories/geo_timeline.jinja2 (hunk 1 above)
Comment (mgallegos3015) [reply templates/archives/stories/geo_timeline.jinja2]: The only source that is CC is the source of the data that is cited. However, I'm unsure how to share the data. For the other sources, does this mean I can just leave them as links?

File: apps/archives/views.py (hunk 2 above)
Comment (mgallegos3015) [reply apps/archives/views.py]: What do you mean? I don't think I understand.

**Comment by mgallegos3015** [comment]
I do hope to finish this soon. I will try to get as much done as possible this week, and I aim to finish before June 15.

**Comment by mscuthbert** [comment]
thank you.  It looks like you'll want to update from Master to complete the conflicts.  Sunday's the last day that any expenses can be charged to the Spring UROP so if you don't think you can complete it, please let me know and Ryaan and I will look through it for what can be merged.  Thanks! M

**Comment by mgallegos3015** [comment]
I was having some issues with files being changed that I had not changed, and I couldn't delete these changes, so I attempted to commit only the files I had changed. Let me know if there are any problems with this.

**Comment by ryaanahmed** [comment]
> I was having some issues with files being changed that I had not changed, and I couldn't delete these changes, so I attempted to commit only the files I had changed. Let me know if there are any problems with this.

Thanks for flagging this up -- just FYI, the files that were changed that you had not changed were part of a merge commit, which would've brought this branch up-to-date with the rest of the project. If you deselect those files, it tells Git _not_ to bring those changes along with this branch, which means that it's now in a state that when merged back into the rest of the project will undo those changes.

Not to worry, though -- I've isolated the commits that you made on a separate branch, and I'm going to merge it with a separate PR. Closing this now. Thanks for the great work!

**Pull request closed without merging**